modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure demonstrates how to deploy application workloads in separate namespaces.
+
+Run the following command to check for existing {istio} instances.
+[source,terminal]
+----
+$ oc get istios
+----
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Add the `istio-discovery=mesh-1` label to ensure that the namespace is discovered by the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio-discovery=mesh-1
+----
+
+. Add the `istio.io/rev=mesh-1` label to the namespace to enable sidecar injection by default while ensuring that pods in this namespace are mapped to the first control plane.
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 istio.io/rev=mesh-1
+----
++
+[NOTE]
+====
+You can verify the `mesh-1` revision name by running the following command:
+[source,terminal]
+----
+$ oc get istiorevisions
+----
+====
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml -n app-ns-1
+----
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml -n app-ns-1
+----
```

Comment:
This can be consolidated into a single command, as you did for the `app-ns-2` and `app-ns-3` namespaces:
```suggestion
[source,terminal]
----
$ oc apply -n app-ns-1 \
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml
----
```